pyflakes/checker.py

```diff
@@ -1662,3 +1662,15 @@ def ANNASSIGN(self, node):
         if node.value:
             # If the assignment has value, handle the *value* now.
             self.handleNode(node.value, node)
+
+    def COMPARE(self, node):
+        literals = (ast.Str, ast.Num)
+        if not PY2:
+            literals += (ast.Bytes,)
+        for op in node.ops:
+            if isinstance(op, (ast.Is, ast.IsNot)):
+                if (isinstance(node.left, literals) or
+                        any(isinstance(right, literals) for right in node.comparators)):
```

Comment:
sorry for the drive by, this produces a false positive (however nonsensical):

```python
# evaluated as x is True and True < 4
x is True < 4
```

there's probably a less-nonsensical false-positive with `enum`s but gotta run to dinner